hrl_ellipsoidal_control/EllipsoidParams params
---
